config/environments/production.rb

```diff
@@ -17,42 +14,34 @@
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = false
 
-  # Attempt to read encrypted secrets from `config/secrets.yml.enc`.
-  # Requires an encryption key in `ENV["RAILS_MASTER_KEY"]` or
-  # `config/secrets.yml.key`.
-  config.read_encrypted_secrets = true
```

Comment:
I don't recall if this was a default or something we added, but should look into that. If it was an old default then removing it makes sense.

Reply:
It's an old default. Also we don't have any `xxx.enc` file (thank god).

I really wish Rails would have opted to go for 12 factor principles rather than invent it's own in house concept where you're supposed to couple server config with your source control...